websites/P/Pandora/presence.ts

```diff
@@ -1,85 +1,87 @@
 const presence = new Presence({
-    clientId: "608109837657702566"
-  }),
-  strings = presence.getStrings({
-    play: "presence.playback.playing",
-    pause: "presence.playback.paused"
-  });
+  clientId: "608109837657702566"
+}),
+strings = presence.getStrings({
+  play: "presence.playback.playing",
+  pause: "presence.playback.paused"
+});
 
 function stripText(element: HTMLElement, id = "None", log = true) {
   if (element && element.firstChild) return element.firstChild.textContent;
   else {
     if (log) {
-      presence.info(
-        `%cPandora%cERROR%c An error occurred while stripping data off the page. Please contact Alanexei on the PreMiD Discord server, and send him a screenshot of this error. ID: ${id}`
+      presence.error(
+        `An error occurred while stripping data off the page. Please contact Alanexei on the PreMiD Discord server, and send him a screenshot of this error. ID: ${id}`
       );
     }
     return null;
   }
 }
 
-let state;
-
 presence.on("UpdateData", async () => {
-  let title: HTMLElement,
-    artist: HTMLElement,
-    smallImageKey,
-    smallImageText,
-    audioTime,
-    audioDuration,
-    details,
-    status,
-    audioElement: HTMLAudioElement = document.querySelector("audio:last-child");
-  audioElement === null
-    ? (audioElement = document.querySelector("audio"))
-    : null;
+  // Define presence data
+  const data: PresenceData = {
+    details: "Browsing...",
+    largeImageKey: "pandora"
+  };
 
-  const audioBar: HTMLElement = document.querySelector(
-    ".Tuner__Audio__NowPlayingHitArea"
-  );
+  // Define whether or not we're currently playing
+  let isPlaying = true;
```

Comment:
Why should this be true by default?

Reply:
Not sure, I was keeping the old functionality. Will set to false.